app/scripts/filters/exists.coffee

```diff
@@ -17,6 +17,8 @@ angular.module('edudashAppFil')
                 for prop, value of input
                     return true;
                 return false
+            if angular.isNumber input && input == NaN
```

Comment:
Hmm. In javascript, `NaN === NaN` is `false`, so the `input == NaN` should _always_ be false here unless coffeescript is doing something weird.

Just checked the [`isNumber`](https://code.angularjs.org/1.3.18/docs/api/ng/function/angular.isNumber) docs:

> This includes the "special" numbers NaN, +Infinity and -Infinity.

> If you wish to exclude these then you can use the native `isFinite' method.

so maybe we should just use `isFinite` here instead ... but do we care about `-Infinity` and `Infinity`?